WORKSPACE

```diff
@@ -4,8 +4,24 @@ load(
     "@d2l_rules_csharp//csharp:defs.bzl",
     "csharp_register_toolchains",
     "csharp_repositories",
+    "import_nuget_package",
 )
 
 csharp_repositories()
 
 csharp_register_toolchains()
+
+# These are some examples.
+# Should this go here? Should we have a WORKSPACE inside csharp/ and one in
+# examples/?
+
+import_nuget_package(
```

Comment:
This case wasn't mentioned in the issue but will work very well for D2L, I think.

In the transitory period with Bazel we end up downloading the packages twice (once for the Bazel and again for the non-Bazel parts of our build) which sucks. Using this we can just wire up the extracted packages that live in `packages/` for now and save time/bandwidth.